apps/src/lab2/views/components/Instructions/ResourcePanel/VersionHistoryRow.tsx

```diff
@@ -0,0 +1,92 @@
+import FontAwesomeV6Icon from '@code-dot-org/component-library/fontAwesomeV6Icon';
+import {RadioButton} from '@code-dot-org/component-library/radioButton';
+import {WithTooltip} from '@code-dot-org/component-library/tooltip';
+import {BodyFourText} from '@code-dot-org/component-library/typography';
+import classNames from 'classnames';
+import React from 'react';
+
+import {INITIAL_VERSION_ID} from '@cdo/apps/lab2/constants';
+import lab2I18n from '@cdo/apps/lab2/locale';
+import {commonI18n} from '@cdo/apps/types/locale';
+
+import moduleStyles from './version-history-panel.module.scss';
+
+interface VersionHistoryRowProps {
+  versionId: string;
+  label: string;
+  isLatest: boolean;
+  isSelected: boolean;
+  comment?: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const VersionHistoryRow: React.FunctionComponent<VersionHistoryRowProps> = ({
+  versionId,
+  label,
+  isLatest,
+  isSelected,
+  comment,
+  onChange,
+}) => {
+  if (isLatest) {
+    label = commonI18n.currentVersion();
+  }
+
+  let rowMarginStyle, ariaLabel;
+  let isBoldtype = true;
+  if (versionId === INITIAL_VERSION_ID) {
+    rowMarginStyle = moduleStyles.isInitialVersionRow;
+  } else if (isLatest) {
+    rowMarginStyle = moduleStyles.isCurrentVersionRow;
```

Comment:
I think in theory the "current version" could be the last committed version, which might be confusing if we don't show the commit details. Something to consider when we get to adding commits.